s3/s3.js

```diff
@@ -0,0 +1,68 @@
+const AWS = require('aws-sdk');
+const { logger } = require('../services');
+
+let s3 = new AWS.S3({ endpoint: 'http://localstack:4572', region: 'us-west-1', s3ForcePathStyle: true });
+
+
+function createBucket(params) {
+  const result = s3.createBucket(params).promise()
+    .then(data => {
+      logger.debug(data);
+      logger.info(`Successful creation of S3 bucket ${params.Bucket}`);
```

Comment:
Change all `logger.info` statements to `logger.debug`. We don't want any logging output other than for debugging purposes. We'll let the consuming modules make determinations on if any events are `info` worthy.

Reply:
✅